backend/.env

```
@@ -1,2 +1,2 @@
-IP_ADD=172.29.240.1
-GPU=NONE #NVIDIA OR NONE
\ No newline at end of file
+IP_ADD=192.168.1.9
+GPU=NVIDIA #NVIDIA OR NONE
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)